chef/site-cookbooks/wca/templates/worldcubeassociation.org.conf.erb

```diff
@@ -30,31 +29,20 @@ server {
   # Using sendfile isn't reliable in VirtualBox. See:
   #  http://wiki.nginx.org/Pitfalls#VirtualBox
   sendfile off;
-<% end %>
-
-<% if @https %>
-  listen 443 ssl;
-  include wca_https.conf;
-<% else %>
-  <% if @rails_env == "development" %>
-    # In development, we're running behind Vagrant's proxy (host port 2331).
-    # If we run nginx on a different port (say 80), then when nginx sends redirects,
-    # they will be to port 80, which doesn't work from outside the vm.
-    listen 2331;
+  # In development, we're running behind Vagrant's proxy (host port 2331).
+  # If we run nginx on a different port (say 80), then when nginx sends redirects,
+  # they will be to port 80, which doesn't work from outside the vm.
+  listen 2331;
   <% else %>
-    listen 80;
-  <% end %>
+  # SSL is handled by the AWS Load Balancer and is only reachable through it
+  listen 80;
```

Comment:
Not sure if this is just the Diff view on GitHub messing up, but the indentation seems off here.

Reply:
it looks off because the listen 80 used to be one if level deeper and there is now just one if